lib/orbit/import/import_certifications.ex

```diff
@@ -0,0 +1,54 @@
+defmodule Orbit.Import.ImportCertifications do
+  alias Orbit.Certification
+  alias Orbit.Import.ImportHelpers
+  alias Orbit.Repo
+
+  @spec import!(Enumerable.t(), Certification.certification_type()) :: :ok
+  def import!(file_lines, certification_type) do
+    file_lines
+    |> Enum.drop_while(fn line -> String.trim(line) != "" end)
+    |> Enum.drop_while(fn line -> String.trim(line) == "" end)
+    |> ImportHelpers.parse_csv(true)
+    |> Enum.map(fn certification ->
+      title = certification["Certifications - Certification Title"]
+      expires_string = certification["Certifications - Certification Period Expiration Date"]
+
+      expires =
+        expires_string |> Timex.parse!("{M}/{D}/{YYYY} {h12}:{m} {AM}") |> NaiveDateTime.to_date()
+
+      rail_line =
+        cond do
+          String.ends_with?(title, "BL") ->
+            :blue
+
+          String.ends_with?(title, "OL") ->
+            :orange
+
+          String.ends_with?(title, "RL") ->
+            :red
+
+          true ->
+            nil
+        end
+
+      %Certification{
+        badge: String.trim_leading(certification["User - User ID"], "0"),
+        type: certification_type,
+        rail_line: rail_line,
+        expires: expires
+      }
+    end)
+    |> Enum.group_by(fn certification -> certification.badge end)
+    |> Enum.map(fn {_badge, certifications} ->
+      Enum.max_by(certifications, fn certification -> certification.expires end, Date)
+    end)
+    |> Enum.each(
+      &Repo.insert(&1,
+        on_conflict: :replace_all,
+        conflict_target: [:badge, :type]
```

Comment:
(Another place where the unique fields are listed, which would need `rail_line` if we decide to do that.)

Reply:
https://github.com/mbta/orbit/pull/203/files#diff-5a8c606e5cebbccd139908748cdfb09e3ad72c2ad41757acb8373c22c4c2a9beR64